src/elements/dialogs/speaker-details.html

```diff
@@ -179,6 +179,10 @@ <h3>{$ speakerDetails.sessions $}</h3>
       _getCloseBtnIcon(isLaptopViewport) {
         return isLaptopViewport ? 'close' : 'arrow-left';
       }
+
+      _hasJobTitle(speaker) {
```

Comment:
Picky... but this is actually more like "`_hasNoJobTitle`"

Reply:
Ohh, got lost in the intermediate edits. Thanks, good catch! Fixed :)